src/TermsAndConditions.tsx

```diff
@@ -0,0 +1,43 @@
+import { PropsWithChildren, useState } from 'react';
+import { Button, Checkbox, Modal, Theme } from 'react-daisyui';
+import { CloseButton } from './components/CloseButton';
+import { useLocalStorage } from './hooks/useLocalStorage';
+
+const EXPIRE_DATE = 30 * 24 * 60 * 60; // 30 days in seconds
+
+const TermsAndConditions = (p: PropsWithChildren) => {
+  const { state, set } = useLocalStorage<string | undefined>({ key: 'termsAndConditions', expire: EXPIRE_DATE });
+  const [checked, setChecked] = useState<boolean>(false);
+
+  const acceptTerms = () => {
+    set('accepted');
+  };
+
+  return state ? (
+    <></>
+  ) : (
+    <Modal open={true} style={{ borderRadius: '5px' }}>
+      <Modal.Header className="text-3xl">T&C</Modal.Header>
+      <Modal.Body>
+        <div className="text-lg mb-4">Alternative links can be found in the docs.</div>
```

Comment:
This text is weird. Was that like that in some design?

Reply:
Is super weird, it was part of the designs.